src/tutorial/scheduling/graphing.rst

```diff
@@ -58,18 +58,11 @@ Example
 Shorthand
 ^^^^^^^^^
 
-Throughout this tutorial we will refer to configuration settings in the following ways:
-
-``[section]``
-   An entire section.
-``[section]key``
-   A specific config item, within a section.
-``[section]key=value``
-   The value of a specific config item, within a section.
-``[section][sub-section]another-key``
-   Note we only use one set of square brackets per section heading when
-   writing on one line like this. In the config file each nested level
-   gets another set of square brackets.
+Throughout the Cylc documentation we use a :ref:`shorthand
+notation<config_item_shorthand>` to refer to
+nested configuration settings on a single line, like this:
+``[section][sub-section]item``.
+
```

Comment:
Could you leave the full section in here as this get included in the tutorial slides which we present from.

Reply:
Ah, OK, I didn't think of that.

I suppose I could move it out into an include-file, to avoid the duplication (potential to get out of sync ...)

Reply:
Or point the user guide at the tutorial, or whatever.

Reply:
Well the file format reference is really where it should be.

Reply:
Try the latest edition (just force-pushed).